docs/en/stack/data-frames/troubleshooting.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[dataframe-troubleshooting]]
+== Troubleshooting {dataframe-transforms}
+[subs="attributes"]
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+Use the information in this section to troubleshoot common problems and find
+answers for frequently asked questions.
+
+include::{stack-repo-dir}/help.asciidoc[tag=get-help]
+
+If you encounter problems with your {dataframe-transforms}, you can gather more
+information from the following files and APIs:
+
+* Lightweight audit messages are stored in `.data-frame-notifications-*`. Search
+by your `transform_id`.
+* The
+{ref}/get-data-frame-transform-stats.html[get {dataframe-transform} statistics API] 
+provides information about the transform status and failures.
+* If the {dataframe-transform} exists as a task, you can use the
```

Comment:
it should exist as a task if it is in either the failed or started state.

Reply:
Thanks, added that sentence.